curriculum/js1/4.js

```diff
@@ -6,8 +6,10 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, result='') => {
```

Comment:
Change a and b to something more descriptive such as num and str. I would also change result to something different, such as newString. It helps to use something that makes sense to you and others.